javascript/demo/demo-node.js

```diff
@@ -1,20 +1,87 @@
-const one = require('oneseismic')
-const http = require('http')
+const one = require("oneseismic");
+const https = require("http");
 
-one().then(mod => {
-    http.get("http://host:port/result/<pid>/stream", {
+/**
+ * To run this demo:
+ *   1. Build the oneseismic library in module mode (aka ONESEISMIC_MODULARIZE=ON)
+ *   2. Let node environment know where to find it (export NODE_PATH=path_to_dir)
+ *   3. Update server, guid and sas_token.
+ *      For demo purpose container must be small and result is expected immediately
+ *      Exchange https with http in "require" if necessary
+ */
+
+one().then((mod) => {
+  erver = "<oneseismic-endpoint>";
+  guid = "<guid>";
+  sas_token = "<sas_token>";
+
+  url = `${server}/graphql?${sas_token}`;
+  const query = `{ cube (id: "${guid}") { sliceByIndex(dim: 0, index: 0) } }`;
+
+  const req = https.request(
+    url,
+    {
+      method: "post",
+      headers: {
+        "Content-type": "application/json",
+      },
+    },
+    (res) => {
+      const { statusCode } = res;
+      if (statusCode !== 200) {
+        let error = new Error(
+          "Request Failed.\n" + `Status Code: ${statusCode}`
+        );
+        console.error(error.message);
```

Comment:
If you don't intend to raise the error, do you even need to create it? You could just log the message directly x2